scenario_configs/eval6/carla_mot/carla_mot_dpatch_undefended.json

```diff
@@ -62,7 +62,6 @@
     "sysconfig": {
         "docker_image": "twosixsarmory/armory-bytetrack",
         "external_github_repo": [
-            "ifzhang/ByteTrack",
```

Comment:
Could you do this on the other two mot configs too?

Reply:
Done.